bramkouwen/Kouwen_Abram_Mariia_Shevchenko_FIP
# Adding css animation

## What does the PR do?
* It creates a css animation for the hero info buttons

## How do you check this PR?
* Pull down the branch
*  See added animation

## Screenshots 
None.
